hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
+  export interface VNodeData {
+    [key: string]: any
+  }
+
+  export interface VNode {
+    tag: string
+    data: VNodeData
+    children: string | VNode | Array<VNode>
+  }
+
+  export type viewFunction<State, Actions> = (
+    state: State,
+    actions: Actions
+  ) => VNode
+  export type emitFunction<Data> = (event: string, data?: Data) => Data
+  export type vnodeFunction = (
+    props: VNodeData,
+    children: string | VNode | Array<VNode>
+  ) => VNode
+
+  export interface Actions<State> {
+    [key: string]:
+      | ((
+          state?: State,
+          actions?: Actions<State>,
+          data?: any,
+          emit?: emitFunction<any>
+        ) => Partial<State> | any)
+      | Actions<State>
+  }
+
+  export interface Events<
+    State,
+    ActionsInterface extends Actions<State> | undefined
+  > {
+    loaded:
+      | ((
```

Comment:
It was not :/
Cause union types : 
`(a: number, b: string) => number | string` 
A function that take `a` as a `number` and `b` as a `string` and return a `number` or a `string`

**is not**
`((a: number, b: string) => number) | string`
A function that take `a` as a `number` and `b` as a `string` and return a `number` ; or a `string`

So let's let them here.